decidim-admin/spec/shared/manage_process_steps_examples.rb

```diff
@@ -97,7 +98,7 @@
     let!(:process_step2) { create(:participatory_process_step, participatory_process: participatory_process) }
 
     before do
-      visit decidim_admin.participatory_process_path(participatory_process)
+      visit current_path
```

Comment:
Oooh, right. It looks like that comment (I need to do this reload in order to show the newly created record.) was posted in the middle of two of my comments so I didn't see it. Sorry for the :trollface:  😞